EIPS/eip-tbd.md

```diff
@@ -0,0 +1,74 @@
+
+1. Changing the value of the GQUADDIVISOR parameter in the ModExp pricing formula to bring its costs more in-line with other precompiles
+2. Modifying the gas pricing formula to better reflect the computational complexity of ModExp operations
+3. Improving the underlying libraries beneath the ModExp Precompile
+4. Any combination of (1), (2), and (3)
+
+We recommend **Option (1)** which provides a large practical improvement to gas estimation while keeping implementation complexity low. Options (2) and (3) could also be implemented and would further improve the gas pricing for a broader range of use cases. Additional data can be provided for options (2) and (3) as desired.
+
+## Motivation
+Modular exponentiation is a foundational arithmetic operation for many cryptographic functions including signatures, VDFs, SNARKs, accumulators, and more. Unfortunately, the ModExp precompile is currently over-priced, making these operations inefficient and expensive. By reducing the cost of this precompile, these cryptographic functions become more practical, enabling improved security, stronger randomness (VDFs), and more.
+
+## Specification
+The current gas pricing formula is defined in EIP-198. This formula divides a ‘computational complexity’ function by a ‘gas conversion’ parameter called ‘GQUADDIVISOR’ to arrive at a gas cost. 
+
+### **Recommended** Option (1): Change value of GQUADDIVISOR
+`GQUADDIVISOR` is set to ‘20’ per EIP-198. We recommend changing the value of this parameter to ‘200’.
+
+### Option (2): Modify ‘computational complexity’ function 
+A proposed ‘complexity’ function can be found at the following [spreadsheet](https://docs.google.com/spreadsheets/d/1Fq3d3wUjGN0R_FX-VPj7TKhCK33ac--P4QXB9MPQ8iw/edit?usp=sharing).
+
+Code defining an improved complexity function can be provided as needed, but this option is not recommended at this time.
+
+### Option (3): Replace libraries used by ModExp precompiles
+ModExp benchmarks for different libraries can be found at the following [spreadsheet](https://docs.google.com/spreadsheets/d/1Fq3d3wUjGN0R_FX-VPj7TKhCK33ac--P4QXB9MPQ8iw/edit?usp=sharing).
+
+While alternative libraries can provide improved performance, this option is not recommended at this time.
+
+## Rationale
+
+### **Recommended** Option (1): Change value of GQUADDIVISOR:
+Changing the value of this parameter from 20 to 200 will reduce the gas cost of this precompile by a factor of 10 with minimal implementation changes. With this change, the cost of the ModExp precompile will have a higher cost (gas/second) than other precompiles such as ECRecover.
+
+![Option 1 Graph](../assets/eip-tbd/GQuad_Change.png?raw=true)
+
+### Option (2): Modify ‘computational complexity’ formula
+An alternative ‘complexity’ function can be found at the following [spreadsheet](https://docs.google.com/spreadsheets/d/1Fq3d3wUjGN0R_FX-VPj7TKhCK33ac--P4QXB9MPQ8iw/edit?usp=sharing).
+
+![Option 2 Graph](../assets/eip-tbd/Complexity_Regression.png?raw=true)
```

Comment:
Not sure `?raw=true` is needed for these links.

Reply:
tried it without and it didn't display. added back in